src/Microsoft.Health.Dicom.Core/Features/Operations/DicomDurableFunctionsHttpClient.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Configs;
+using Microsoft.Health.Dicom.Core.Messages.Operations;
+using Microsoft.Health.Dicom.Core.Models.Operations;
+using Microsoft.Net.Http.Headers;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Dicom.Core.Features.Operations
+{
+    internal class DicomDurableFunctionsHttpClient : IDicomOperationsClient
+    {
+        private readonly HttpClient _client;
+        private readonly OperationsConfiguration _config;
+        private static readonly JsonSerializerSettings JsonSettings = new JsonSerializerSettings
+        {
+            DateTimeZoneHandling = DateTimeZoneHandling.Utc,
```

Comment:
We're still serializing using the "round-trip" format, but it means that:
1. We'll treat all unspecified `DateTime` objects as UTC when serializing
2. We'll convert local `DateTime` objects to UTC when serializing

This way we'll consistently output the values in UTC as oppose to possibly serializing values in different time zones (although I am fairly certain the durable functions API returns UTC too). It would all be equally valid to return different time zones in the "round-trip" format, but as a web API hosted in some DC I don't think we want to ever return arbitrary local time zones based on where we happen to be deployed.